src/core_plugins/console/index.js

```diff
@@ -150,14 +151,13 @@ module.exports = function (kibana) {
         path: '/api/console/api_server',
         method: ['GET', 'POST'],
         handler: function (req, reply) {
-          let server = require('./api_server/server');
-          let { sense_version, apis } = req.query;
+          const { senseVersion, apis } = req.query;
```

Comment:
Originally we expected `sense_version` in the response, but now we're expecting `senseVersion`. Do we need to update any server-side code to make a corresponding change?